### DISCUSSION TITLE
Executing KeyDB EXPIREMEMBER command

### DISCUSSION THREAD

**Discussion by binaryfire** [phpredis/phpredis#2572] [Q&A] [answered]
Hi all

I'm using KeyDB as a drop-in replacement for Redis and want to use their `EXPIREMEMBER` command. The command is unique to KeyDB. Is there a way of doing that with phpredis?

Thanks!

**Comment by michael-grunder** [comment, marked as answer]
Yes, you can use `rawCommand`:

```php
$redis->rawCommand('EXPIREMEMBER', 'hash', 'field', 123);
```

We can also implement it natively before PhpRedis 6.1.1

Edit: Implemented #2573
